docs/docs/apis/providers/backend-services-api.md

```diff
@@ -0,0 +1,169 @@
+- `AssetService` - Native token and NFT data
+- `BlockService` - Block information and navigation
+- `EpochService` - Epoch and staking information
+
+## Supported Backend Services
+
+| Provider | Module | Features | Status |
+|----------|--------|----------|--------|
+| **Blockfrost** | `cardano-client-backend-blockfrost` | Complete API coverage, high reliability | ✅ Stable |
+| **Koios** | `cardano-client-backend-koios` | REST API, community-driven | ✅ Stable |
+| **Ogmios** | `cardano-client-backend-ogmios` | WebSocket, real-time updates | ✅ Stable |
+
+## Usage Examples
+
+### Blockfrost Integration
+
+Initialize Blockfrost backend service with API key:
+
+```java
+// Initialize Blockfrost service
+BackendService backendService = new BFBackendService(
+    Constants.BLOCKFROST_TESTNET_URL, 
+    "your-project-id"
+);
+
+// Access different services
+TransactionService transactionService = backendService.getTransactionService();
+UtxoService utxoService = backendService.getUtxoService();
+AddressService addressService = backendService.getAddressService();
+AssetService assetService = backendService.getAssetService();
+
+// Use with QuickTx
+QuickTxBuilder quickTxBuilder = new QuickTxBuilder(backendService);
+```
+
+### Koios Integration
+
+Initialize Koios backend service for community-driven API:
+
+```java
+// Initialize Koios service
+BackendService backendService = new KoiosBackendService(
+    "https://api.koios.rest/api/v1"
+);
+
+// Access services
+TransactionService transactionService = backendService.getTransactionService();
+UtxoService utxoService = backendService.getUtxoService();
+AddressService addressService = backendService.getAddressService();
+
+// Query address information
+List<AddressContent> addressInfo = addressService.getAddressInfo("addr1...");
+System.out.println("Address balance: " + addressInfo.get(0).getAmount());
+```
+
+### Ogmios Integration
+
+Initialize Ogmios backend service for WebSocket-based API:
+
+```java
+// Initialize Ogmios service
+BackendService backendService = new OgmiosBackendService(
+    "ws://localhost:1337"
+);
+
+// Access services
+TransactionService transactionService = backendService.getTransactionService();
+UtxoService utxoService = backendService.getUtxoService();
+
+// Submit transaction
+Result<String> result = transactionService.submitTransaction(transactionCbor);
+if (result.isSuccessful()) {
+    System.out.println("Transaction submitted: " + result.getValue());
+}
+```
+
+## Advanced Usage
+
+### Provider Switching
+
+Switch between different providers based on requirements:
+
+```java
+public class BackendServiceFactory {
+    
+    public static BackendService createService(String provider, String endpoint, String apiKey) {
+        switch (provider.toLowerCase()) {
+            case "blockfrost":
+                return new BFBackendService(endpoint, apiKey);
+            case "koios":
+                return new KoiosBackendService(endpoint);
+            case "ogmios":
```

Comment:
We can add `kupomios` instead of `ogmios`. In that case, `KupmiosBackendService` will be used.